package/src/util.ts

```diff
@@ -0,0 +1,161 @@
+import type { Rank, ValorantMap } from "./types/general.js"
```

Comment:
Why are you importing javascript files here?

Also this files doesn't even exist.